packages/vanilla/src/sass/components/accordion/_index.scss

```diff
@@ -1,148 +1,127 @@
-//@use '../../base/colors' as *;
-@use '../../typography' as *;
-@use './states' as *;
-//@use '../../tokens/font' as *;
-
-.cbp-accordion {
-  margin: 24px 0;
+.cbp-accordion-group {
+  margin-top: var(--cbp-space-6x);
```

Comment:
Why not still use the shorthand than 2 separate properties? It's shorter.

Reply:
By default it doesn't add left/right margins so it just needs to be set on the top/bottom. There is a top margin in the mockups so we can leave it in there and see if a future mock up with other patterns makes it look bad.